grails-app/controllers/au/org/ala/spatial/service/ReportController.groovy

```diff
@@ -25,18 +27,25 @@ class ReportController {
 
         //render taskService.report() as JSON
     }
-    def task(){
+
+    def task() {
+        boolean includeAll = false
+        if (params.containsKey("includeAll") && !params.includeAll) {
```

Comment:
Hard coded "includeAll"? Case sensitive?

Reply:
Case sensitive on purpose.  The default and preferred way is to excludes some ALA members

Reply:
Ok